src/main/java/core/basesyntax/service/GenerateReportService.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import core.basesyntax.dao.FruitDao;
+
+public interface GenerateReportService {
+
+    String generateReport(FruitDao fruitDao);
```

Comment:
Pass FruitDao to constructor
```suggestion
public interface GenerateReportService {
    String generateReport();
```